geth/api/backend_jail_test.go

```diff
@@ -780,17 +784,30 @@ func (s *BackendTestSuite) TestJailVMPersistence() {
 	for _, tc := range testCases {
 		wg.Add(1)
 		go func(tc testCase) {
+			defer wg.Done() // ensure we don't forget it
+
 			s.T().Logf("CALL START: %v %v", tc.command, tc.params)
 			response := jailInstance.Call(testChatID, tc.command, tc.params)
 			if err := tc.validator(response); err != nil {
 				s.T().Errorf("failed test validation: %v, err: %v", tc.command, err)
 			}
 			s.T().Logf("CALL END: %v %v", tc.command, tc.params)
-
-			wg.Done()
 		}(tc)
 	}
 
-	common.PanicAfter(10*time.Second, nil, "test timed out")
+	common.PanicAfter(60*time.Second, nil, "test timed out")
 	wg.Wait()
+
+	// Validate total.
+	cell, err := jailInstance.GetJailCell(testChatID)
+	require.NoError(err)
+
+	totalOtto, err := cell.Get("total")
+	require.NoError(err)
+
+	total, err := totalOtto.ToFloat()
+	require.NoError(err)
+
+	s.T().Log(total)
+	require.Equal(0.840003, math.Round(total, 6))
```

Comment:
I find it best to use difference in delta as well. As a better means of gauging the difference in values for floats.